docs/src/main/asciidoc/config-yaml.adoc

```diff
@@ -170,32 +328,66 @@ display:
     factor: ${x.factor}
 ----
 
-Note that you can reference nested properties using the `.` (dot) separator as in `${x.factor}`.
+[NOTE]
+====
+You can reference nested properties by using the `.` (dot) separator, as in `{x.factor}`.
+====
+
+.Additional resources
+
+* For more information about property expressions, see xref:con_property-expressions_{context}[Property expressions].
+// * For an example of property expressions using a properties file, see link:{URL_CONFIGURATION_QUARKUS}#ref_example-usage-of-property-expressions_quarkus-configuration-guide[Example usage of property expressions]. // TBD update and enable this ^ link later when this guide becomes available
+
+// DELETE meta:include::modules/con_external-application-yaml-file-for-configuring-properties-at-runtime.adoc[leveloffset=+1]
+// CREATE meta:include::modules/proc_configure-properties-at-run-time-external-yaml.adoc[leveloffset=+1]
+[id="proc_configure-properties-at-run-time-external-yaml_{context}"]
+== Configure properties at run time
 
-== External application.yaml file
+To configure properties at run time, you can create an external configuration file by placing an `application.yaml` file in the `config` directory.
 
-The `application.yaml` file may also be placed in `config/application.yaml` to specialize the runtime configuration. The
-file has to be present in the root of the working directory relative to the Quarkus application runner:
+When you run the application, the values from the `config/application.yaml` file override those of identical properties in `resources/application.yaml` file.
 
-[source, text]
+.Prerequisites
+
+include::{includes}/prerequisites.adoc[]
+* The `quarkus-config-yaml` extension as a dependency in your project's `pom.xml` or `build.gradle` file.
+
+.Procedure
+
+. Add an `application.yaml` file to the `config` directory.
+
+. Verify that the `config/application.yaml` file is in the root of the working directory relative to the {project-name} application runner, as shown in the following example:
++
+[source,properties,options="nowrap"]
 ----
-.
 ├── config
 │    └── application.yaml
 ├── my-app-runner
 ----
 
-The values from this file override any values from the regular `application.yaml` file if exists.
+.Additional resources
+
+* xref:proc_enabling-yaml-configuration_{context}[Enabling YAML configuration].
+
+//include::modules/proc_managing-configuration-key-conflicts.adoc[leveloffset=+1]
+[id="proc_managing-configuration-key-conflicts_{context}"]
+== Managing configuration key conflicts
 
-== Configuration key conflicts
+Structured formats such as YAML only support a subset of the possible configuration namespace.
+The following procedure shows how to resolve a conflict between two configuration properties, `quarkus.http.cors` and `quarkus.http.cors.methods`, where one property is the prefix of another.
```

Comment:
I'm not sure about the rules and maybe this needs to be discussed at a higher level but we are adding a ton of procedures/prerequisites and this adds a lot of noise when you are trying to find some information.

Reply:
Please feel free to strike any information that doesn't add value for our customers.
In future projects, rather than trying to merge the community and product content, I'll adopt an "additive" process. That is, I'll ask the SME to identify which product content they want to migrate to the community docs first.

Reply:
Or, if you feel like that's too much trouble, I can toss out this PR and we start over with the proposed "additive" process instead.